src/extensions/typography/controls.js

```diff
@@ -18,6 +19,13 @@ class Controls extends Component {
 			attributes,
 		} = this.props;
 
+		if ( ! coblocksBlockData.disableTypography ) {
```

Comment:
This should be `( coblocksBlockData.disableTypography )`. in the current state the typography controls are always disabled.

Reply:
I see now that this was related to the administrators filter. This effectively always disables the extension unless a user has applied a filter to allow administrators or specific pages access.

Reply:
Fixed.